data/abilities.js

```diff
@@ -1297,11 +1293,8 @@ let BattleAbilities = {
 		onResidualOrder: 5,
 		onResidualSubOrder: 1,
 		onResidual(pokemon) {
-			if (pokemon.side.active.length === 1) {
-				return;
-			}
-			for (const allyActive of pokemon.side.active) {
-				if (allyActive && allyActive.hp && this.isAdjacent(pokemon, allyActive) && allyActive.status && this.randomChance(3, 10)) {
+			for (const allyActive of pokemon.allies(true)) {
```

Comment:
This was renamed to `nearbyAllies` and will need a rebase. Also I still like `adjacentAllies` better.

Reply:
I renamed it back, but apparently I missed some of these.